### DISCUSSION TITLE
Is it possible to remove certain pages

### DISCUSSION THREAD

**Discussion by DariusL** [kudrykv/latex-yearly-planner#145] [Q&A]
There are pages I never use like reflections and I'd like a smaller PDF. The Kindle struggles a bit, especially towards the end of the year. Is this something currently configurable? I could generate the PDFs myself. If not, I'd appreciate this feature.

**Comment by leggs11** [comment]
I was looking for the same and not finding an option I found a barbaric method that worked:
- After running the "plannergen" command with your desired config, but before using "xeletex" command, go to the "out" folder and delete the "dailyreflections" or pages you want to delete.
- Run the "xelatex" function, it will report some errors but successfully compile the PDF
- You now have the PDF without the reflect pages, it dropped the filesize from around 1900kb to 1500kb without the reflections.
Hope this helps

**Reply by DariusL** [reply]
Thanks for the suggestion, but I already got it removed through even more barbaric methods. Used a PDF editor to remove some of the pages. Even earlier pages like quarterly notes were removed with no problems for the later pages or links.
